drivers/hmis/app/models/hmis/search/assessment_search.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Search::AssessmentSearch
```

Comment:
for consistency, can we call filters "filter" instead of "search"?  `filter/assessment_filter.rb`